CHANGELOG.md

```diff
@@ -464,6 +464,8 @@ Key points:
 
 - #1568: Make DispatcherSchema merkelized again [ECR-3810]
 
+- #1592: Fix node free after re-enable consensus [ERC-3111]
```

Comment:
"Fix node free" doesn't sound quite correct in English.

Reply:
Mac autocorrect thinks different.